.evergreen/config.yml

```diff
@@ -2025,18 +2025,25 @@ axes:
         variables:
           SETDEFAULTENCODING: "cp1251"
 
-  - id: requireApiVersion
-    display_name: "requireApiVersion"
+  - id: versionedApi
+    display_name: "versionedApi"
     values:
+      # Test against a cluster with requireApiVersion=1.
       - id: "requireApiVersion1"
         display_name: "requireApiVersion1"
-        tags: [ "requireApiVersion_tag" ]
+        tags: [ "versionedApi_tag" ]
         variables:
           # REQUIRE_API_VERSION is set to make drivers-evergreen-tools
           # start a cluster with the requireApiVersion parameter.
           REQUIRE_API_VERSION: "1"
           # MONGODB_API_VERSION is the apiVersion to use in the test suite.
           MONGODB_API_VERSION: "1"
+      # Test against a cluster acceptAPIVersion2 but without requireApiVersion=0.
+      - id: "acceptAPIVersion2"
+        display_name: "acceptAPIVersion2"
+        tags: [ "versionedApi_tag" ]
+        variables:
+          ORCHESTRATION_FILE: "versioned-api-testing.json"
```

Comment:
I reworked this section slightly because in one case we want to test two different cases:
- against a cluster with requireApiVersion=1 and with `MONGODB_API_VERSION`
- against a cluster without requireApiVersion without `MONGODB_API_VERSION` but with acceptAPIVersion2.